privacy-snapshot.json

```diff
@@ -9,7 +9,9 @@
   "bafybeidxfmwycgzcp4v2togflpqh2gnibuexjy4m4qqwxp7nh3jx5zlh4y.ipfs.dweb.link",
   "cdnjs.cloudflare.com",
   "chainid.network",
+  "configuration.dev.metamask-institutional.io",
```

Comment:
I had to add this after merging in develop, it currently breaks builds on develop

Reply:
Likely an MMI config change for test builds: MMI_CONFIGURATION_SERVICE_URL